opengrok-indexer/pom.xml

```diff
@@ -194,6 +194,11 @@ Portions Copyright (c) 2017-2019, Chris Fraire <cfraire@me.com>.
             <artifactId>jackson-annotations</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.dropwizard.metrics</groupId>
+            <artifactId>metrics-core</artifactId>
```

Comment:
On https://dropwizard.io/en/stable/manual/core.html I see that `metrics-core` package depends on whole lot of other packages. Some of them are already delivered. How does it work e.g. with Jersey/Jackson version ?

Reply:
Good catch. Maven resolves this somehow, I believe by including the newest version. However, I can specify that these dependencies are already provided and the ones from Dropwizard should be ignored.

Reply:
As long as both work.. :-)